src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,383 @@
+
+    .pf-c-sequence__content-title {
+      color: var(--pf-c-sequence__content-title--hover--Color);
+    }
+
+    .pf-c-sequence__number {
+      min-width: var(--pf-c-sequence__number--hover--MinWidth);
+      max-height: var(--pf-c-sequence__number--hover--MaxHeight);
+      font-size: var(--pf-c-sequence__number--hover--FontSize);
+      color: var(--pf-c-sequence__number--hover--Color);
+
+      &::before {
+        border: var(--pf-c-sequence__number--BorderWeight) solid var(--pf-c-sequence__number--hover--Color);
+      }
+    }
+  }
+
+  &.pf-m-active,
+  &:active {
+    border-bottom: var(--pf-c-sequence__link--BorderBottomHeight) solid var(--pf-c-sequence__link--active--BorderBottomColor);
+
+    .pf-c-sequence__number {
+      min-width: var(--pf-c-sequence__number--active--MinWidth);
+      max-height: var(--pf-c-sequence__number--active--MaxHeight);
+      font-size: var(--pf-c-sequence__number--active--FontSize);
+      color: var(--pf-c-sequence__number--active--Color);
+
+      &::before {
+        border: var(--pf-c-sequence__number--BorderWeight) solid var(--pf-c-sequence__number--active--Color);
+      }
+    }
+
+    .pf-c-sequence__content-title {
+      color: var(--pf-c-sequence__content-title--active--Color);
+    }
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-bottom: var(--pf-c-sequence__link--BorderBottomHeight) solid var(--pf-c-sequence__link--focus--BorderBottomColor);
+
+    .pf-c-sequence__number {
+      min-width: var(--pf-c-sequence__number--focus--MinWidth);
+      max-height: var(--pf-c-sequence__number--focus--MaxHeight);
+      font-size: var(--pf-c-sequence__number--focus--FontSize);
+      color: var(--pf-c-sequence__number--focus--Color);
+
+      &::before {
+        border: var(--pf-c-sequence__number--BorderWeight) solid var(--pf-c-sequence__number--focus--Color);
+      }
+    }
+
+    .pf-c-sequence__content-title {
+      font-weight: var(--pf-c-sequence__content-title--focus--FontWeight);
+      color: var(--pf-c-sequence__content-title--focus--Color);
+    }
+  }
+
+  &.pf-m-current {
+    border-bottom: var(--pf-c-sequence__link--m-current--BorderBottomHeight) solid var(--pf-c-sequence__link--m-current--BorderBottomColor);
+
+    .pf-c-sequence__number {
+      min-width: var(--pf-c-sequence__number--m-current--MinWidth);
+      max-height: var(--pf-c-sequence__number--m-current--MaxHeight);
+      font-size: var(--pf-c-sequence__number--m-current--FontSize);
+      color: var(--pf-c-sequence__number--m-current--Color);
+
+      &::before {
+        border: var(--pf-c-sequence__number--BorderWeight) solid var(--pf-c-sequence__number--m-current--Color);
+      }
+    }
+
+    .pf-c-sequence__content-title {
+      font-weight: var(--pf-c-sequence__content-title--m-current--FontWeight);
+      color: var(--pf-c-sequence__content-title--m-current--Color);
+    }
+  }
+}
+
+// Override button styles
+button {
```

Comment:
This is a global change, why is it here?

Reply:
Thanks for picking that up. Is it correct to move it to _base.scss?